### ISSUE TITLE
mb.YTPlayer v. 3.4.0 doesn't center correctly

### ISSUE THREAD

**Issue by Saintpid** [pupunzi/jquery.mb.YTPlayer#419]
I've selected center, center and that has previously centered the video nicely, since the latest update the film is pushed to the right side of the window.

**Comment by Saintpid** [comment]
www.saintpid.se

**Comment by Saintpid** [comment]
Thanks. I'll look into that.

**Comment by Saintpid** [comment]
Hello again, I think I removed or fixed every javascript error I could find. I still get errors but those seem to be caused by Chrome Extensions since I get no errors in an incognito window. The background video still doesn't center correctly. Do you have any more ideas that could be the cause, the former version worked?

**Comment by pupunzi** [comment]
Hi, can you try changing

  line 2588:    
      
```
            default:
              if (vid.width > win.width)
                vid.marginLeft = -((vid.width - win.width) / 2) + (abundance / 2);
              break;
```
with:
           
```
            default:
              if (vid.width > win.width)
                vid.marginLeft = -(((vid.width + abundance) - win.width ) / 2) ;
              break;
```
on the jquery.mb.YTPlayer.js

**Comment by Saintpid** [comment]
That snippet of code moved the movie too far to the left instead of too far to the right, it didn't center it. I tried to remove the '+ abundance' and that centered it correctly (but surely messed up something else of course).

`            default:
              if (vid.width > win.width)
                vid.marginLeft = -(((vid.width) - win.width ) / 2) ;
              break;`

**Comment by Saintpid** [comment]
I just saw that you posted an update (3.4.1), it works as expected again. Thank you!

**Comment by pupunzi** [comment]
@andreafrison Saintpid refers to the Wordpress plugin.
The fix has not been released yet. If you download the master it contains the fix for that otherwise I'll publish the 3.3.0 in the next days.
